src/adapters/kalao-avalanche/abi.json

```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "inputs": [{ "internalType": "uint256", "name": "_tokenId", "type": "uint256" }],
+        "name": "getSeller",
+        "outputs": [{ "internalType": "address", "name": "", "type": "address" }],
+        "stateMutability": "nonpayable",
+        "type": "function"
+    },
+    {
+        "inputs": [{ "internalType": "uint256", "name": "_blockNumber", "type": "uint256" }],
+        "name": "getPrice",
+        "outputs": [{ "internalType": "uint256", "name": "", "type": "uint256" }],
+        "stateMutability": "nonpayable",
+        "type": "function"
+    }
+]
```

Comment:
[memo]
kalao abi is not published, so I decompiled it and read decompiled code.

https://snowtrace.io/bytecode-decompiler?a=0x11ac3118309a7215c6d87c7c396e2df333ae3a9c